recipes-browser/cog/cog_0.12.0.bb

```diff
@@ -0,0 +1,17 @@
+require cog.inc
+require conf/include/devupstream.inc
+
+SRC_URI[sha256sum] = "aecf546d7b0645119d79559c5574cb5eb68364fc8409dfbd47a4920bd1f221bc"
+
+# Avalilable from 0.10.X
```

Comment:
Same here. By removing the 0.6.0 recipe we could move this flag to the cog.inc.

Reply:
I think that would be a mistake, only 0.12.x supports the `COG_PLATFORM_WL` option. Actually I would argue that we have to move `COG_PLATFORM_FDO` *outside* of `cog.inc` and into the `.bb` files of the versions that support that form of the option.

Let me re-spin things a bit and submit a new version with my proposed changes—if you still think everything should be in the `cog.inc` file regardless of whether a version supports a given option or not, let me know afterwards and I'll change things back to be that way.

Reply:
this comment is about the 'PACKAGECONFIG[headless]'.

In general, all the common options for the available versions should be on the .inc file. Since We are going to keep only the 0.12 recipe I think all the flags defined,  even those in origin defined for devupstream and also part of 0.12, should be in the .inc file.